km/km_decode.c

```diff
@@ -820,7 +820,16 @@ static void decode_opcode(km_vcpu_t* vcpu, x86_instruction_t* ins)
       return;
    }
 
-   km_warnx("KM intruction decode: Uninterpreted Opcode=0x%x", opcode);
+   uint64_t pc = vcpu->regs.rip;
+   km_warnx("KM intruction decode: PC 0x%x, Uninterpreted Instruction=%02x %02x %02x %02x %02x "
```

Comment:
not enough bytes for some instructions. But we will get the prefix which is what we need.

Reply:
There should be enough for prefix, opcode, modrm byte, and the sib byte.